app/components/Accordion.tsx

```
@@ -0,0 +1,166 @@
+    const isActive = isItemActive(itemId);
+
+    const defaultIcon = (
+        <svg
+            className={cn("w-5 h-5 transition-transform duration-200", {
+                "rotate-180": isActive,
+            })}
+            fill="none"
+            stroke="#98A2B3"
+            viewBox="0 0 24 24"
+            xmlns="http://www.w3.org/2000/svg"
+        >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)